Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -214,7 +216,7 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
       .take(first: 1)
       .observeValues { appOptions in
         _ = AppEnvironment.current.facebookAppDelegate.application(
-          appOptions.application,
+          appOptions.application ?? UIApplication.shared,
```

Comment:
The Facebook ApplicationDelegate interface updated to receive this as non-optional. Coalescing here is the quickest fix, an alternative would be to the mock this as a whole different type throughout the codebase which seemed like more work than its worth. It turns out this value is `nil` when tests run 🤷‍♂